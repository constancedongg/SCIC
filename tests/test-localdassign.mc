/*locally declare and assign variables*/

bool func larger(int a, int b) {
    string ss = "inside compare func";
    printl(ss);
    return a > b;
}

int func main() {
    float '{A} I = 1.2;
    printl("Second line in main");
    bool islarger = larger(1, 3);  
    printb(islarger);  
    return 0;
}