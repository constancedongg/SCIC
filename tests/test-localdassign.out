Second line in main
inside compare func
0